tex/generic/pgf/basiclayer/pgfcoretransformations.code.tex

```diff
@@ -466,6 +466,10 @@
 % }
 %
 \def\pgftransformlineattime#1#2#3{%
+  \let\pgf@tempaa\pgf@pt@aa
+  \let\pgf@tempab\pgf@pt@ab
+  \let\pgf@tempba\pgf@pt@ba
+  \let\pgf@tempbb\pgf@pt@bb
```

Comment:
The three pair of additions to `\pgftransformlineattime`, `\pgftransformarcaxesattime`, and `\pgftransformcurveattime` are so similar that maybe two new macros abstracting them correspondingly can be provided.

Reply:
Good idea, unfortunately `\pgftransformlineattime` uses `\pgf@xc`/`\pgf@yc` and `\pgf@x`/`\pgf@y` isn't save to use inbetween. I would just refactor that so that it uses x/y instead of xc/yc. (Too big of a change?)

That said, there's a lot more that they have in common.